lib/http/serve/serve_test_FuzzTestObjectBadRange.go

```diff
@@ -0,0 +1,33 @@
+package serve
+
+import (
+	"net/http/httptest"
+	"os"
+	"secsys/gout-transformation/pkg/transstruct"
```

Comment:
Where does this package come from?

Reply:
I'm curious as well (arrived here, after we received a similar pull request in our repository https://github.com/moby/moby/pull/46421). I'm not sure if these PRs are "legit", also looking at the author's profile activity

<img width="791" alt="Screenshot 2023-09-07 at 16 08 09" src="https://github.com/rclone/rclone/assets/1804568/5a3dc234-fbe8-43cf-ad90-c468af055b4b">

Reply:
Hmm, curious. The PR itself looks like code but I have no idea how it might work!

The `{insert repo here}` PRs are odd too.

I'm hoping we might hear from the author to explain.

Reply:
Yup! So "Fuzzing" code (to what I've seen) usually looks a bit odd (on purpose).

I'm a big "+1" on having more fuzzing on projects; just the PRs from this contributor looked (at first glance) "dodgy", so I thought let me at least comment!

If you're interested in Fuzzing, @AdamKorcz has been helping various projects to set up (and can be trusted). We're still not doing as much fuzzing in Moby as would be desired, but that's really "us" to blame (time and all that)

Reply:
@ncw we discussed the contribution in our maintainers call, and after looking, we considered that these are _NOT_ legit contributions;

- golang package names MUST have a `.` (domain name) in the first element, so the `secsys/gout-transformation/pkg/transstruct` is invalid
- The fuzzing functions do _not_ match the correct signature
- The fuzzing functions must be in a `_test.go` file, and both contributions do _not_ (and would break production code)

I'm reporting the account for abuse, but you could decide to already ban them from your org.